src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -94,10 +104,36 @@ private static void OnProcessExit(object sender, EventArgs e)
             }
         }
 
+        private void LogEventRaised(NativeInterop.Core sender, LogEventArgs args)
+        {
+            if (_logger != null)
+            {
+                using LogEventWrapper logEventWrapper = new LogEventWrapper(this);
+                LogLevel msalLogLevel = LogLevelMap[args.LogLevel];
+                if (_logger.IsLoggingEnabled(msalLogLevel))
+                {
+                    string msgWithPii = string.Empty;
+                    string msgNoPii = string.Empty;
+
+                    if (_logger.PiiLoggingEnabled)
+                    {
+                        msgWithPii = args.Message;
+                    }
+                    else
+                    {
+                        msgNoPii = args.Message;
+                    }
```

Comment:
args.Message is the same thing. See my comment about PII.

Reply:
Updated as there will be no Pii in the first release